src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Modules/CoreEndpointModule.cs

```diff
@@ -11,9 +11,10 @@ public class CoreEndpointModule : IEndpointModule
 {
     public void MapEndpoints(IEndpointRouteBuilder endpoints)
     {
-        endpoints.MapPost("/data/{**dmsPath}", Upsert);
-        endpoints.MapGet("/data/{**dmsPath}", Get);
-        endpoints.MapPut("/data/{**dmsPath}", UpdateById);
-        endpoints.MapDelete("/data/{**dmsPath}", DeleteById);
+        bool enforceAuthorization = endpoints.ServiceProvider.GetRequiredService<IConfiguration>().GetValue<bool>("IdentitySettings:EnforceAuthorization");
```

Comment:
Shouldn't we have a shared function somewhere so that we don't need to repeat `.GetValue<bool>("IdentitySettings:EnforceAuthorization")` and on `WebApplicationBuilderExtensions` line 129?